doc/tests/manual/protocol.md

```diff
@@ -297,6 +297,35 @@
 - Test Result:
 - Notes:
 
+## TC-018 - Edit seed
+
+- Description: Edit seed.
+- Preconditions:
+  - User is on the view seed page.
+- Test Steps:
+  1. Press Button "Edit seed"
+  2. Change an attribute of the seed.
+  3. Submit the form.
+  4. Repeat steps 1 through 3 for every seed attribute.
+- Expected Result:
+  - The seed attributes updated successfully.
+- Actual Result:
+- Test Result:
+- Notes:
+
+## TC-019 - Delete seed
+
+- Description: Edit seed.
+- Preconditions:
+  - User is on the view seed page.
+- Test Steps:
+  1. Press Button "Delete seed"
```

Comment:
Same problem but different reaction (via Toast instead of popup). Please use Toasts everywhere.